doc/source/api.rst

```diff
@@ -809,6 +809,7 @@ Binary operator functions
    DataFrame.eq
    DataFrame.combine
    DataFrame.combine_first
+   DataFrame.round
```

Comment:
I don't think this is the right place (it is not a binary operator). But not sure what the good place is ..

Maybe 'Computations / Descriptive Stats'